stac_pydantic/item.py

```diff
@@ -20,7 +20,7 @@ class ItemProperties(StacCommonMetadata):
     """
 
     # Overide the datetime field to be required
-    datetime: Optional[UtcDatetime]
+    datetime: Optional[UtcDatetime] = Field(...)
```

Comment:
Note that the [spec says](https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#datetime) that `start_datetime` and `end_datetime` must be set if `datetime` is null ... not sure if that is enforced currently or not.

Reply:
it's enforced in the validator